conans/client/build/cmake_flags.py

```diff
@@ -52,7 +53,7 @@ def get_generator(settings):
                     '16': '16 2019'}
         base = "Visual Studio %s" % _visuals.get(compiler_version,
                                                  "UnknownVersion %s" % compiler_version)
-        if Version(compiler_version) < "16":
+        if os_host == "Windows" and Version(compiler_version) < "16":
```

Comment:
I don't know if "WindowsStore" would be affected by this if. Should we use `if os_host != "WindowsCE" and ...` ?

Reply:
According to this CMake tests the cmake generator doesn't include "ARM" or "Win64" suffix:
https://gitlab.kitware.com/cmake/cmake/commit/22282d69316eaf6e4a2b50af2ca7585a58f00d28

I'll test "WindowsStore" and, if necessary, request a separate PR.